README.md

```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
 
 For the main reference implementation of the ILP stack, see [Interledger.js](https://github.com/interledgerjs).
 
-## Index
-
-Here is a list of the main Interledger RFCs:
+## Interledger Overview and Explanatory Docs
```

Comment:
Right now I just put them in numerical order within the groups, but that's not necessarily the best ordering